packages/synapse-react-client/src/components/styled/LightTooltip.tsx

```diff
@@ -26,8 +26,8 @@ export const LightTooltip: StyledComponent<TooltipProps> = styled(
     },
     color: theme.palette.background.paper,
   },
-  [`& .${linkClasses.root}`]: {
-    color: theme.palette.primary.main,
+  [`& .${tooltipClasses.tooltip} .${linkClasses.root}`]: {
+    color: `${theme.palette.primary.main}`,
   },
 }))
```

Comment:
Increase the specificity of this rule to ensure it actually applies. It currently has the same specificity as the rule that makes Links in Tooltips white, since our tooltips have a dark background by default. This is inappropriate for the `LightTooltip`, so we use the `main` color